include/dem/particle_particle_contact_force.h

```diff
@@ -437,56 +437,47 @@ class ParticleParticleContactForce
     const unsigned int particle_two_type =
       particle_two_properties[PropertiesIndex::type];
 
+
     // Calculation of normal and tangential spring and dashpot constants
     // using particle properties
     double normal_spring_constant =
       1.0667 * sqrt(this->effective_radius) *
       this->effective_youngs_modulus[vec_particle_type_index(
         particle_one_type, particle_two_properties[PropertiesIndex::type])] *
-      pow((0.9375 * this->effective_mass * normal_relative_velocity_value *
-           normal_relative_velocity_value /
+      pow((0.9375 * this->effective_mass * 1.0 *
+           1.0 / // Characteristic velocity is set to 1.0
```

Comment:
I would keep a variable called `characteristic velocity` = 1.0 and then explain that it is 1.0 because...

Reply:
That's a good idea. can be a static const double = 1.0 for now.

Reply:
" Characteristic velocity is set to 1.0 so that the normal and tangential spring_constant remain constant throughout a simulation."